README.md

```diff
@@ -41,12 +41,13 @@ developers.
   a [development preview](https://github.com/kblaschke/frontend-sdl2/releases/tag/2.0-windows-pre1))
 - [Steam](https://store.steampowered.com/app/1358800/projectM_Music_Visualizer/) (Same as standalone development
   preview)
-- [Windows Store](https://www.microsoft.com/store/apps/9NDCVH0VCWJN) (Old 3.1.12 release )
+- [Windows Store](https://www.microsoft.com/store/apps/9NDCVH0VCWJN) (Old 3.1.12 release)
```

Comment:
Since there's reports saying that this release no longer works on XBox, we may be better off to remove the link. We probably can't easily persuade Microsoft to remove it from the store, but at least no longer claim it to be an official release as it's probably more damaging than doing good.